### DISCUSSION TITLE
How can i use the specified gpu

### DISCUSSION THREAD

**Discussion by liqiran23** [open-edge-platform/anomalib#1883] [Q&A] [answered]

**Comment by blaz-r** [comment, marked as answer]
Hi, you can pass any arguments to Engine that Lightning trainer takes. Here are the [Engine](https://anomalib.readthedocs.io/en/latest/markdown/guides/reference/engine/index.html#module-anomalib.engine) docs and here is the [GPU training](https://lightning.ai/docs/pytorch/stable/accelerators/gpu_basic.html) guide from Lightning.
